engine/action_contact.go

```diff
@@ -16,9 +17,9 @@ func (c *Client) UpsertContact(ctx context.Context, ctcFName, ctcPaymail, reques
 		return nil, err
 	}
 
-	pmSrvnt := &PaymailServant{
-		cs: c.Cachestore(),
-		pc: c.PaymailClient(),
+	pmSrvnt, err := paymailclient.NewServiceClient(c.Cachestore(), c.PaymailClient())
+	if err != nil {
+		panic(err)
```

Comment:
I'm wondering if the `panic` should be inside the `NewServiceClient` function.

Reply:
hmm... maybe I will move creation of this service to be responsibility of engine and then we can panic on startup.

I need to make additional changes then, sorry 🙈

Reply:
Good idea 👍